aplib.net-demo/Assets/New Terrain.asset.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
+guid: dd50548435531d94d8085a80b439bdf2
+NativeFormatImporter:
+  externalObjects: {}
+  mainObjectFileID: 15600000
+  userData: 
+  assetBundleName: 
+  assetBundleVariant: 
```

Comment:
What is this asset? It should not be in the root folder, and if it is not necessary, it should be deleted.

Reply:
deleted